recipes/nbtutor/build.sh

```diff
@@ -0,0 +1,19 @@
+conda install --yes nodejs
+# FIXME: This is an ugly hack. nodejs only gets installed and linked in the
+# first build env and doesn't get linked in following build envs
```

Comment:
It seems like since this recipe was last run, our conda-build was updated. Would you mind merging/rebasing from master and pushing it up again?